### DISCUSSION TITLE
Should I separate .obsidian between devices?

### DISCUSSION THREAD

**Discussion by AppleFritter58** [Vinzent03/obsidian-git#232] [Q&A]
I use git plugin between iOS, Windows, and Mac. The concern is separating the .obsidian folders (.obsidian for laptops, and .obsidian/mobile for my iOS devices). This often results in my mobile devices not keeping the same settings as my PC devices, and I want that to happen. What are the chances of issues occurring if I switch back to using one .obsidian file though?

**Comment by Vinzent03** [comment]
If you configure your `.gitignore` correctly, I don't think there are any issues. Just make sure that often changed files are ignored (#231).
Plugins that are only compatible with desktop are automatically disabled on mobile.
